### PR TITLE
Build Awesome Books using ES6 and Modules (rm)

### PR DESCRIPTION
**Pull request by Jord4N-AC**
In this project, I restructured my Awesome books app to make it more organized by using modules, I also applied the ES6 syntax.

### Project Requirements:

- **Setup project:**
  - Create a new directory and initialize it with git
  - Use the npm init -y command to create a package.json file
  - Create the entry point for your JavaScript code called index.js
  - Create an index.css file for styles
  - Create the main index.html file and link index.js and index.css files
  - Create a directory called modules

- **Re-create the Awesome books app following these rules:**
  - Use ES6 syntax
  - Divide my code into modules. Save each module in a separate file in the modules dir. Import modules in the index.js file (more modules the better)
  - Keep all functionalities of the app without errors
  - Refactor my methods and functions to arrow functions
  - Use let and const in a correct way

- **Display the current date below the navigation bar:**
  - Use a library Luxon
  - Download the script and save it to the project
  - Import it as a module
  - Use this tool with little externally curated support
  - 

### General Requirements:
- Check linter errors
- Use the correct GitHub flow
- Document my work in a professional way
- Follow best practices for HTML, CSS & JavaScript
- Open a pull request and ask for a review

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -15,5 +15,8 @@ BookObject.loadBooks();
 window.addEventListener('hashchange', showSection);
 
 // Date
-const now = DateTime.now();
-date.innerHTML = now.toLocaleString(DateTime.DATETIME_MED_WITH_SECONDS);
+function showTime() {
+  const now = DateTime.now();
+  date.innerHTML = now.toLocaleString(DateTime.DATETIME_MED_WITH_SECONDS);
+}
+setInterval(showTime, 1000);
```
Comment (Whoistolu) [inline index.js]: - [x] [REQUIRED] Project works fine, however, kindly ensure you use an arrow function here and in every other place you declared a regular function.  Arrow functions are

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @Jord4N-AC 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [x]  _Check the inline comments under the review for sections to improve._
- [x] [REQUIRED] In your README file, kindly state the set of instructions needed to run the project, such as : cloning, switching to the right branch and running `npm i`

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — APPROVED** [review]
Hi @Jord4N-AC , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Jord4N-AC** [comment]
@OLIPLICHE thank you so much for your review

**Review by GzimAsani — COMMENTED** [review]
Hi @Jord4N-AC , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that you have already merged.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**